orchagent/fdborch.cpp

```diff
@@ -1138,6 +1138,37 @@ void FdbOrch::flushFDBEntries(sai_object_id_t bridge_port_oid,
         }
     }
 }
+void FdbOrch::flushFdbByVlan(const string &alias)
+{
+    sai_status_t status;
+    swss::Port vlan;
+    sai_attribute_t vlan_attr[2];
+
+    if (!m_portsOrch->getPort(alias, vlan))
+    {
+        return;
```

Comment:
Can we add an Error message here printing the alias details.

Reply:
Done